setup.sh

```diff
@@ -28,52 +28,65 @@ check_install_pkg() {
             echo "Unknown error installing $pkg"
             echo $install_output
         fi
-
     fi
 }
-check_install_pip(){
+check_install_pip2(){
 
 	pip_ver=$($pip --version | grep --line-buffered "^Version")
 	if [ -n "$pip_ver" ]; then
 		echo "$pip_ver found"
 
 	else
 		echo "pip not found, Installing... "
-        mkdir -p tools
-        curl https://bootstrap.pypa.io/get-pip.py -o tools/get-pip.py
-		install_output=$(python tools/get-pip.py --user)
+		install_output=$(sudo apt install python-pip)
 	fi
 
 	if [ -n "$(echo $install_output | $grep_success)" ]; then
             pip_ver=$(pip --version | sed 's/from/at/g')
             echo "Successfully installed $pip_ver"
         else
-            echo "Unknown error installing pip"
+            echo "Unknown error installing pip"             
         fi
 }
+check_install_pip3(){
+
+	pip_ver=$($pip --version | grep --line-buffered "^Version")
+	if [ -n "$pip_ver" ]; then
+		echo "$pip_ver found"
 
+	else
+		echo "pip not found, Installing... "
+		install_output=$(sudo apt install python3-pip)
+	fi
 
+	if [ -n "$(echo $install_output | $grep_success)" ]; then
+            pip_ver=$(pip --version | sed 's/from/at/g')
+            echo "Successfully installed $pip_ver"
+        else
+            echo "Unknown error installing pip"
+        fi
+}
 
 #Check python2 version, if not available install
 python2_ver=$(python --version 2>&1 | $grep_ver)
```

Comment:
As noted in a previous review comment. I think you can retire python 2 support as it's at end of life...

Reply:
Removed python2 support from [setup.sh](https://github.com/direct-state-transfer/dst-doc/pull/15/commits/17c920773c6174f11c5e8a5ce82ef8ffb60a892e)